docs/user/contacts.md

```diff
@@ -0,0 +1,53 @@
+# Contacts
+
+## Table of Contents
+
+- [Bug Report](#Bug-Report)
+- [Proposal](#Proposal)
+- [Feature Request](#Feature-Request)
+- [Security Issue Report](#Security-Issue-Report)
+
+### Bug Report
+
+A bug is a demonstrable problem which produce incorrect result or to behave in unintended ways.
+Bug reports are helpful for developers who maintain the Vald project.
+
+A good bug report shouldn’t leave others needing to ask you for more information constantly.
+Please try to write as detail as possible in your bug report.
+
+Things to check before reporting a bug:
+- Check your environment
+Bug may occur depending on different version of Vald, Kubernetes and etc.
+
+- Identify the bug
+Developers need to reproduce the bug in the developers environment.
+So please identify the bug and clarify how it occurs.
+Please also please clarify the expected behavior too.
+
+- Use the Github issue search
+The bug may be reported before.
+So please check the Github issue before reporting.
+
+### Proposal
+
+The Vald project’s development process is design-driven. 
+The significant change to the library, or the architecture will be discussed first.
+
+We may need a detail documentation before you proposal is implemented.
+Your proposal will be reviewed and discussed and decide whether it is accepted or declined.
+
+### Feature Request
+
+Feature request is welcome.
+
+Before opening an issue, please make sure your idea fits the project.
+You can make a strong claim to convince the project maintainers to accept your feature request.
```

Comment:
`claim` does not suit this document 😅 
My suggestion is here. How do you think, also @kevindiu ?
```suggestion
You can require strongly to convince the project maintainers to accept your feature request.
```